src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-top: 46px;
+  }
+
+  &__image {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__name {
+    padding-left: $card-content-side-paddings;
+    padding-right: $card-content-side-paddings;
+    padding-top: 40px;
+    margin: 0;
+
+    @extend %normal-text;
+    line-height: 18px;
+    color: $default-color;
+  }
+
+  &__product-code {
+    padding: 4px 0 $card-content-side-paddings $card-content-side-paddings;
+    margin: 0;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $seccondary-text-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 24px;
+  }
+
+  &__price-word {
+    padding-left: $card-content-side-paddings;
+    margin: 0;
+
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $seccondary-text-color;
+  }
+
+  &__price-number {
+    padding-right: $card-content-side-paddings;
+    padding-left: 85px;
+    margin: 0;
+
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $default-color;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-top: 0;
+    margin: 16px;
+    width: 166px;
```

Comment:
do not hardcode width and height of the button, use paddings for height and width 100%, it will fit if you put paddings on the card